packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1716,6 +1716,32 @@ export class Vector3 {
         return Scalar.NormalizeRadians(angle);
     }
 
+    /**
+     * Gets the angles between two points needed to face the target from the start

```

Comment:
Since the method return a Vector3 this cannot be desribed as angles. Change comments to

```
 /**
     * Gets the rotation that aligns the roll axis (Y) to the line joining two points and stores it in the ref Vector3
     * @param start the starting point
     * @param target the target point
     * @param ref the vector3 to store the result
     * @returns the rotation in the form (pitch, yaw, 0)
     */
```